sql/core/src/test/scala/org/apache/spark/sql/DataFrameSuite.scala

```diff
@@ -830,6 +830,15 @@ class DataFrameSuite extends QueryTest
     ("David", 60, 192),
     ("Amy", 24, 180)).toDF("name", "age", "height")
 
+  private lazy val person3: DataFrame = Seq(
```

Comment:
Could you move this test into `DataFrameStatSuite`?

Reply:
All the other Dataset#summary related tests are here, so might be best to keep them all in one place.  It is a little tricky to find the tests sometimes, so might be good to address the test organization as part of a larger refactoring.